components/cam/cime_config/config_component.xml

```diff
@@ -294,6 +296,7 @@
       <value compset="2000_CAM5%SP2"     >2000_cam5_av1c-04p2-MMF-2mom</value>
       <value compset="2000_CAM5%MMF1"    >2000_cam5_av1c-04p2-MMF-1mom</value>
       <value compset="2000_CAM5%MMF2"    >2000_cam5_av1c-04p2-MMF-2mom</value>
+      <value compset="_CAM5%MMF1-RCE" >RCEMIP_EAMv1</value>
```

Comment:
Is there a reason you don't use MMF[12] here? Assuming it's only valid for 1-moment, but is there harm in leaving the config open to 2-moment?

Reply:
this should be [12], I'll correct it